src/bentoml/_internal/io_descriptors/json.py

```diff
@@ -6,6 +6,8 @@
 import dataclasses
 
 import attr
+from typing_extensions import TypedDict
+from typing_extensions import is_typeddict
```

Comment:
Let's do a check here, 

```suggestion

if sys.version_info[:2] >= (3, 8):
	from typing import TypedDict, is_typeddict
else:
	from typing_extensions import TypedDict, is_typeddict
```

Reply:
Please also add typing_extensions to `pyproject.toml`.